.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,34 @@
+name: Deployment
+
+on:
+  push:
+    branches: [production]
```

Comment:
since only the production branch will be used, it is not necessary to have a list of branches to target

Reply:
Done